policyengine_us/parameters/gov/states/mo/tax/income/rates.yaml

```diff
@@ -1,70 +1,99 @@
-description: The tax rates for individual residents in Missouri. Thresholds are <=.
+description: The tax rates for individual residents in Missouri.
+# threshold value is bottom of the tax bracket; thresholds are <=
 brackets:
   - rate:
       2019-01-01: 0.015
     threshold:
       2019-01-01: 0
-    # NB: Technically, there's a further exemption for the first $100 of income or so
-    # but this is essentially a rounding choice that we ignore ($1-2 impact).
   - rate:
       2019-01-01: 0.02
     threshold:
       2019-01-01: 1_053
       2020-01-01: 1_073
       2021-01-01: 1_088
+      2022-01-01: 1_121
+      2023-01-01: 1_000
```

Comment:
Won't the the 2022 value be closer to correct? Or did I miss brackets resetting this year in the legal code?

Reply:
@MaxGhenis said:
> Won't the the 2022 value [1_121] be closer to correct?

I don't understand the question.  The $1121 value is straight out of the 2022 instructions.

Reply:
Right, won't the 2023 value be higher than $1,121? If so I think we should approximate it by leaving it at $1,121 rather than resetting to $1,000. Separately we need to CPI-index many parameters like these in the future.

Reply:
I interpreted [the law](https://revisor.mo.gov/main/OneSection.aspx?section=143.011&bid=51511&hl=) as only adjusting the top rate (no brackets) in 2023. Am I missing something?

> 2. (1)  Notwithstanding the provisions of subsection 1 of this section to the contrary, beginning with the 2023 calendar year, the top rate of tax pursuant to subsection 1 of this section shall be four and ninety-five hundredths percent.
> 
>   (2)  The modification of tax rates made pursuant to this subsection shall apply only to tax years that begin on or after January 1, 2023.
> 
>   (3)  The director of the department of revenue shall, by rule, adjust the tax table provided in subsection 1 of this section to effectuate the provisions of this subsection.  The top remaining rate of tax shall apply to all income in excess of seven thousand dollars, as adjusted pursuant to subsection 5 of this section.
> 
> ...
> 
>   5.  Beginning with the 2017 calendar year, the brackets of Missouri taxable income identified in subsection 1 of this section shall be adjusted annually by the percent increase in inflation.  The director shall publish such brackets annually beginning on or after October 1, 2016.  Modifications to the brackets shall take effect on January first of each calendar year and shall apply to tax years beginning on or after the effective date of the new brackets.

Reply:
@MaxGhenis, so I guess your view is the following table (which is mentioned in the recent reform table of tax brackets and rates is the one from 2017:

<img width="1920" alt="Screenshot 2023-02-13 at 9 35 51 PM" src="https://user-images.githubusercontent.com/12170745/218625205-96b53fb2-2715-471c-bef7-7912dc66f0cd.png">

I'm not sure why the new legislation would repeat the old table, but I'll defer to you assessment of the Missouri tax law.   I'll revise this pull request to include only **known** tax-bracket values from the forms through 2022.